benchmarks/synchrobench/src/main/java/com/yahoo/oak/synchrobench/contention/benchmark/ThreadLoopOak.java

```diff
@@ -95,7 +95,8 @@ public void run() {
 
         MyBuffer key = new MyBuffer(Parameters.keySize);
 
-        Integer newInt = -1;
+        Integer newInt = (Parameters.keyDistribution == Parameters.KeyDist.RANDOM) ? -1 : Parameters.size;
```

Comment:
The conditional assignment is redundant. You can simply assign `Parameters.size` because in the case of random distribution, it will override anyway.
Please add some comment here explaining why we assign `Parameters.size` and not simply zero.

Reply:
fixed